packages/repository/src/__tests__/unit/model/model.unit.ts

```diff
@@ -46,8 +46,7 @@ describe('model', () => {
   const flexibleDef = new ModelDefinition('Flexible');
   flexibleDef
     .addProperty('id', {type: 'string', id: true})
-    .addProperty('createdAt', Date)
-    .addSetting('hiddenProperties', ['createdAt'])
+    .addProperty('createdAt', {type: Date, hidden: true})
```

Comment:
Please don't modify existing tests, we want to keep this test to ensure `hiddenProperties` setting is still working as expected. Add a new test and/or define a new test model instead.

In my experience, it's best to avoid sharing test model that's very specific and define such model inside the test that is using it, see `it('includes navigational properties in JSON` below for an example to follow.